python/datasources/data_source.py

```diff
@@ -45,26 +45,33 @@ def write_to_bq_table(self, dataset: str, gcs_bucket: str, filename: str, table_
         chunked_frame = gcs_to_bq_util.load_csv_as_dataframe(
             gcs_bucket, filename, chunksize=1000)
 
-        # Replace spaces and dashes with underscores in column name and make all characters
-        # in column names lower case.
         for chunk in chunked_frame:
-            chunk.rename(columns=lambda col: (
-                col
-                .lower()
-                .strip()
-                .replace(' ', '_')
-                .replace('-', '_')
-                .replace(':', '_')
-                .replace('&', '_')
-                .replace("\n", '_')
-                .replace("\t", '_')
-                .replace('(', '_')
-                .replace(')', '_')
-                .replace('=', 'eq')
-            ), inplace=True)
+            self.clean_frame_column_names(chunk)
             gcs_to_bq_util.append_dataframe_to_bq(
                 chunk, dataset, table_name, project=project)
 
+    def clean_frame_column_names(self, frame):
+        """ Replaces unfitting BigQuery column
+        characters and makes them all coumn names
```

Comment:
nit: "makes all column names"

super nit: I think this comment could fit on 2 lines

Reply:
Done